relayer/client.go

```diff
@@ -108,18 +108,22 @@ func CreateClient(ctx context.Context, src, dst *Chain, srcUpdateHeader, dstUpda
 
 	// Query the trusting period for dst and retry if the query fails
 	var tp time.Duration
-	if err := retry.Do(func() error {
-		var err error
-		tp, err = dst.GetTrustingPeriod(ctx)
-		if err != nil {
-			return fmt.Errorf("failed to get trusting period for chain{%s}: %w", dst.ChainID(), err)
-		}
-		if tp == 0 {
-			return fmt.Errorf("chain %s reported invalid zero trusting period", dst.ChainID())
+	if customClientTrustingPeriod == 0 {
```

Comment:
I would initialize this as `tp := customClientTrustingPeriod` so that we don't need an `else` condition on the if-statement.